docs/docs/APIGuide/Layers/Recurrent-Layers.md

```diff
@@ -9,7 +9,7 @@ val module = Recurrent()
 module = Recurrent()
 ```
 
-Recurrent module is a container of rnn cells. Different types of rnn cells can be added using add() function.  
+Recurrent module is a container of rnn cells. Different types of rnn cells can be added using add() function. Don't support multiCell for performance issue. Use several Recurrent(cell) instead.  
```

Comment:
How do you enforce that?

Reply:
In recurrent.add(), it will check the cell type, if it's multirnncell, it will throw exception and ask user use sequential().add(Recurrent.add(cell))... instead